cartographer/mapping_2d/sparse_pose_graph/constraint_builder.cc

```diff
@@ -60,6 +60,16 @@ ConstraintBuilder::~ConstraintBuilder() {
   CHECK(when_done_ == nullptr);
 }
 
+/**
```

Comment:
Oh, good to know. :) to be honest I'm pretty new to C/C++ so i didn't know about this convention. Now that I see the comments there mine seam to be mostly redundant.
But maybe we can convert the existing ones to doxygen, so an IDE will see them?